test/modules/DecentSablierStreamManagement.test.ts

```diff
@@ -38,10 +46,10 @@ describe('DecentSablierStreamManagement', () => {
   let mockHats: MockHats;
   let mockHatsAddress: string;
 
-  let decentHats: DecentHats_0_1_0;
+  let decentHats: DecentHatsCreationModule;
```

Comment:
Might wanna rename ´decentHats´ -> ´decentHatsCreationModule´

Reply:
Eh it's a test, whatever

Reply:
But yeah, I did updated it